.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 80,
```

Comment:
The notion print width of 80 columns is so old school.
[This is where it started](https://en.wikipedia.org/wiki/Punched_card)

Reply:
😂 I got that line from the .prettierrc file in salesforcedx-vscode.  I'll remove it here but then we should also remove it there too.